CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Cromwell Change Log
 
+## 53 Release Notes
+
+### Bug fixes
+
+* Wired through support for WDL optional output files on the PAPI v2 backend.
```

Comment:
```suggestion
* Added support for WDL optional output files on the PAPI v2 backend.
```

Reply:
Does this still belong under 'Bug fixes' if it's an addition?

Reply:
IMO it's new functionality not a bug fix